src/rcc.rs

```diff
@@ -0,0 +1,242 @@
+use camino::Utf8Path;
```

Comment:
Since we only use this for testing, let's move it to the test folder

Reply:
Done